awx/ui_next/src/screens/ExecutionEnvironment/shared/ExecutionEnvironmentForm.jsx

```diff
@@ -124,8 +126,15 @@ function ExecutionEnvironmentFormFields({
 
       <CredentialLookup
         label={i18n._(t`Registry credential`)}
+        credentialTypeKind="registry"
+        helperTextInvalid={credentialMeta.error}
+        isValid={!credentialMeta.touched || !credentialMeta.error}
+        onBlur={() => credentialHelpers.setTouched()}
         onChange={onCredentialChange}
         value={credentialField.value}
+        tooltip={i18n._(
+          t`Credential to authenticate with a protected container registry. Must be of type "Container Registry".`
```

Comment:
@nixocio

Since we're _only_ showing container registry credentials, do we really need to warn the user about other types of credentials? Presumably they wouldn't even be able to select other types.

Reply:
@jakemcdermott, I looked at the container groups tooltip, there we also filter by a specific type of credential, and display the following message.

```Credential to authenticate with Kubernetes or OpenShift. Must be of type "Kubernetes/OpenShift API Bearer Token". If left blank, the underlying Pod's service account will be used.```

We can easily update this one if we decide that is redundant. Just let me know.

Reply:
Let's leave the tooltips in but remove the "Must be of type" warning.

Reply:
@jakemcdermott, done. I will merge this one @tiagodread already approved. Thank you both.